scripts/zones/Batallia_Downs_[S]/mobs/Lycopodium.lua

```diff
@@ -0,0 +1,17 @@
+-----------------------------------
+-- Area: Batallia Downs [S]
+--  Mob: Lycopodium
+-----------------------------------
+mixins = {require("scripts/mixins/families/lycopodium")}
+-----------------------------------
+
+local entity = {}
+
+entity.onMobInitialize = function(mob)
+    mob:setMobMod(xi.mobMod.ALWAYS_AGGRO, 1)
```

Comment:
Why not either set them agro via their pool or bake this into their mixin script?

The extant example of this in SEA (Hpemde) use the pool value, if I recall correctly but it could easily be an onSpawn in the mixin as well.

Reply:
I am not familiar with something you can set in their pool to make them always aggro, please let me know if there is. They should always follow players regardless of the player's level so without this, at a certain point you won't be followed by the mandies if you out level their aggro range.

The reason I placed this in the onMobInitiialize is that the ALWAYS_AGGRO mod was not properly being set on spawn through the mixin. I assume the mixin doesn't get applied to the mandies until after they have spawned so the intial round of lycopodiums spawned into the world would not properly have the mod set.

Reply:
I'll look into why it's not working at the mixin level, it should. I can always move this there later, so marking this convo resolved and letting it through per mob. A comment about it in file would be good though :+1: 

Was mainly looking for why because I figured there had to be a problem for you to do it this way.

Things not applying on the initial spawn is a problem in itself - that 1st round of spawns should be firing off all the "on spawn" things after their onInit is done.